cypher-shell/src/integration-test/java/org/neo4j/shell/MainIntegrationTest.java

```diff
@@ -418,12 +420,9 @@ public void doesNotStartWhenDefaultDatabaseUnavailableIfInteractive() throws Exc
             shell.disconnect();
 
             // Should get exception that database is unavailable when trying to connect
-            exception.expect(TransientException.class);
-            exception.expectMessage("Database 'neo4j' is unavailable");
             main.connectMaybeInteractively(shell, connectionConfig, true, true);
```

Comment:
If we are doing it with `catch` we should probably add a `fail` here otherwise test will pass if no exception is thrown